5_evaluate.R

```diff
@@ -51,9 +62,18 @@ p5 <- list(
   # Match GCM predictions to observations
   # map over obs_for_eval_groups (so parallelizable on Tallgrass)
   tar_target(p5_gcm_pred_obs,
-             match_pred_obs(preds_file = sprintf('3_extract/out/GLM_%s_%s.feather', unique(p5_gcm_obs_for_eval_groups$site_id), p1_gcm_names),
-                            eval_obs = p5_gcm_obs_for_eval_groups, lake_depth = p5_gcm_obs_depths, driver = p1_gcm_names),
-             pattern = cross(map(p5_gcm_obs_for_eval_groups, p5_gcm_obs_depths), p1_gcm_names)),
+             {
+               tar_assert_identical(unique(p5_gcm_export_tibble_groups$site_id), unique(p5_gcm_obs_for_eval_groups$site_id),
```

Comment:
Just adding a teeny bit more context here

```suggestion
               # Each branch has one site id, but must match predictions from each of the 6 GCM drivers
               tar_assert_identical(unique(p5_gcm_export_tibble_groups$site_id), unique(p5_gcm_obs_for_eval_groups$site_id),
```